### PR TITLE
kokoro: enable more tests

### PR DESCRIPTION
**Pull request by tbpg**
Part of #251.

### CODE REVIEW

File: .kokoro/ruby-2.5.cfg [hunk 1]
```diff
@@ -5,3 +5,8 @@ env_vars: {
     key: "TRAMPOLINE_IMAGE"
     value: "gcr.io/cloud-devrel-kokoro-resources/ruby-2.5"
 }
+
+env_vars: {
+    key: "RUN_ALL_TESTS"
+    value: "1"
+}
```
Comment (frankyn) [inline .kokoro/ruby-2.5.cfg]: Add new line

File: .kokoro/system_tests.sh [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 source $KOKORO_GFILE_DIR/secrets.sh
 
+# Get a project from the project pool.
```
Comment (frankyn) [inline .kokoro/system_tests.sh]: Is this tool documented externally? Or internally at least?

File: spec/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+
+for i in {0..9}; do
```
Comment (frankyn) [inline spec/readme.md]: This looks incomplete.

File: .kokoro/ruby-2.5.cfg (hunk 1 above)
Comment (tbpg) [reply .kokoro/ruby-2.5.cfg]: Done.

File: .kokoro/system_tests.sh (hunk 2 above)
Comment (tbpg) [reply .kokoro/system_tests.sh]: Added a link to the implementation.

File: spec/readme.md (hunk 3 above)
Comment (tbpg) [reply spec/readme.md]: Done. May need more next time we add a project - may have missed some details.

**Pull request merged**